meson.build

```diff
@@ -71,6 +71,43 @@ if get_option('disable_gds_backend')
     add_project_arguments('-DDISABLE_GDS_BACKEND', language: 'cpp')
 endif
 
+# Logging
+
+# Determine the actual log level based on user choice or build type
+log_level_opt = get_option('log_level')
+buildtype = get_option('buildtype')
+
+actual_log_level = log_level_opt
+if actual_log_level == 'auto'
+  if buildtype == 'debug'
+    actual_log_level = 'trace'
+  else # release, debugoptimized, or plain
+    actual_log_level = 'warning'
```

Comment:
sorry for confusion let's also have info in the release build, so at least some important information is available